ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/apps/[externalID]/syncs/Sync/Sync.tsx

```diff
@@ -46,6 +48,24 @@ export function Sync({ externalAppID, syncID }: Props) {
   const { app } = syncRes.data.environment;
   const { sync } = syncRes.data;
 
+  let functions = null;
+  if (sync.status === SyncStatus.Success) {
+    functions = (
+      <FunctionList
+        removedFunctions={sync.removedFunctions}
+        syncedFunctions={sync.syncedFunctions}
+      />
+    );
+    // TODO: Replace with SyncStatus.duplicate after we deploy the API changes
+  } else if (sync.status === 'duplicate') {
+    functions = (
+      <Alert severity="info">
+        This sync is a duplicate. Please navigate to the previous successful sync to view its
```

Comment:
I'm not sure I get what a duplicate is/ how it happens, this would benefit from more details or a link to the explanation in docs.